docs/content/preview/reference/drivers/csharp/postgres-npgsql-reference.md

```diff
@@ -25,61 +25,56 @@ showAsideToc: true
+Npgsql is an open source ADO.NET Data Provider for PostgreSQL; it allows programs written in C#, Visual Basic, and F# to access the YugabyteDB server.
 
-## Quick Start
+## Quick start
 
-Learn how to establish a connection to YugabyteDB database and begin simple CRUD operations using the steps in [Build an Application](/preview/quick-start/build-apps/csharp/ysql) in the Quick Start section.
+Learn how to establish a connection to YugabyteDB database and begin CRUD operations using the steps from [Build a C# application](../../../../quick-start/build-apps/csharp/ysql).
 
-## Download the Driver Dependency
+## Download the driver dependency
 
-If you are using Visual Studio IDE, follow the below steps:
-1. Open your Project Solution View
-1. Right-click on **Packages** and click **Add Packages**
+If you are using Visual Studio IDE, add the Npgsql package to your project as follows:
+
+1. Right-click **Dependencies** and choose **Manage Nuget Packages**
 1. Search for `Npgsql` and click **Add Package**
 
-To add Npgsql package to your project, when not using an IDE, use the `dotnet` command:
+To add the Npgsql package to your project when not using an IDE, use the following `dotnet` command:
+
 ```csharp
-dotnet add package Npgsql 
-``` 
+dotnet add package Npgsql
+```
+
 or any of the other methods mentioned on the [nuget page](https://www.nuget.org/packages/Npgsql/) for Npgsql.
 
 ## Fundamentals
 
-Learn how to perform the common tasks required for Java App development using the PostgreSQL JDBC driver
-
-<!-- * [Connect to YugabyteDB Database](postgres-jdbc-fundamentals/#connect-to-yugabytedb-database)
-* [Configure SSL/TLS](postgres-jdbc-fundamentals/#configure-ssl-tls)
-* [Create Table](/postgres-jdbc-fundamentals/#create-table)
-* [Read and Write Queries](/postgres-jdbc-fundamentals/#read-and-write-queries) -->
+Learn how to perform common tasks required for C# application development using the Npgsql driver.
 
-### Connect to YugabyteDB Database
+### Connect to YugabyteDB database
 
-After setting up the dependenices, we implement the C# client application that uses the Npgsql driver to connect to your YugabyteDB cluster and run query on the sample data.
+After setting up the dependencies, implement the C# client application that uses the Npgsql driver to connect to your YugabyteDB cluster and run a query on the sample data.
 
-We will import Npgsql and use the `NpgsqlConnection` class for getting connection object for the YugabyteDB Database which can be used for performing DDLs and DMLs against the database.
+Import Npgsql and use the `NpgsqlConnection` class to create the connection object to perform DDLs and DMLs against the database.
 
-Example URL for connecting to YugabyteDB can be seen below.
+Example connection string parameter (connStringBuilder) for connecting to YugabyteDB is as follows:
```

Comment:
I'd drop in the changes from the same section on the drivers page - table first, then example string

Reply:
got it , re-ordered the tables on both the pages.